diff --git a/Model/Indexer/Category/Product/Action/Full.php  b/Model/Indexer/Category/Product/Action/Full.php
index 5a5dced8..95285306 100644
--- a/Model/Indexer/Category/Product/Action/Full.php
+++ b/Model/Indexer/Category/Product/Action/Full.php
@@ -14,6 +14,16 @@ class Full extends \Magento\Catalog\Model\Indexer\Category\Product\AbstractActio
      */
     protected $useTempTable = false;

+    /**
+     * Still the best option to fix the 500 cap on categories.
+     * this is fixed in new version of magento
+     * @return bool
+     */
+    protected function isRangingNeeded()
+    {
+        return false;
+    }
+
     /**
      * Refresh entities index
      *
