diff --git a/Model/Rule.php b/Model/Rule.php
index 06422c4e..1f944e7f 100644
--- a/Model/Rule.php
+++ b/Model/Rule.php
@@ -283,7 +283,7 @@ class Rule extends \Magento\Rule\Model\AbstractModel
             )->setUsagePerCustomer(
                 $this->getUsesPerCustomer() ? $this->getUsesPerCustomer() : null
             )->setExpirationDate(
-                $this->getToDate()
+                null //$this->getToDate() github.com/magento/magento2/issues/6122
             )->save();
         } else {
             $this->getPrimaryCoupon()->delete();
