From: Navarr Barnier <me@navarr.me>
Date: Mon, 16 Jan 2016 11:15:00 +0500
Subject: [PATCH] Fixes LiteMage attempting to hit 127.0.0.1 during CLI Cache Flush

---

diff --git a/Observer/FlushCacheByCli.php b/Observer/FlushCacheByCli.php
--- Observer/FlushCacheByCli.php
+++ Observer/FlushCacheByCli.php
@@ -99,7 +99,7 @@
 		$server_ip = '127.0.0.1' ; //$helper->getConf($helper::CFG_WARMUP_SERVER_IP, $helper::CFG_WARMUP);
 		$base = $this->url->getBaseUrl();
-		if ($server_ip) {
+		if (false && $server_ip) {
 			$pattern = "/:\/\/([^\/^:]+)(\/|:)?/";
 			if (preg_match($pattern, $base, $m)) {
 				$domain = $m[1];
